Gemfile.lock

```diff
@@ -10,16 +10,16 @@ GIT
 PATH
   remote: .
   specs:
-    cassandra-utils (0.2.4)
-      daemon_runner (~> 0.2)
+    cassandra-utils (0.0.1)
```

Comment:
I don't have a local VERSION file, so it reverted to the 0.0.1 version. As far as I know, this shouldn't break anything. @athompson-r7, any thoughts on this?